app/client/cypress/integration/national/national.spec.js

```diff
@@ -0,0 +1,188 @@
+describe('State Links', () => {
+  beforeEach(() => {
+    cy.visit('/national');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('not.exist');
+  });
+
+  it('Clicking the "National Drinking Water" tab changes the tab and does not update the Url', () => {
+    // verify tab changed correctly
+    cy.findByText('National Drinking Water').click();
+    cy.get('h2')
+      .filter(':visible')
+      .contains('Explore National Drinking Water Information')
+      .should('exist');
```

Comment:
For this I'd suggest doing something similar to what discussed during the testing meeting and put a "data-testid" on each panel. Then you can do 

```suggestion
    cy.findByTestId('<the-id>')
       .findByText('Explore National Drinking Water Information')
       .should('be.visible');
```